ngraph/src/ngraph/graph_util.cpp

```diff
@@ -305,11 +306,14 @@ std::vector<std::shared_ptr<ngraph::Node>>
                 }
             }
             auto cloned_node = node->copy_with_new_inputs(cloned_args, cloned_dependencies);
-            if (node->get_friendly_name() != node->get_name())
+            // There is a friendly name for this node so copy it
+            cloned_node->set_friendly_name(node->get_friendly_name());
+            //  TODO: workaround for shape inference, delete it after fix
```

Comment:
do we have issue for this case?

Reply:
It is a copy paste from specialize_function.

@itikhono Do we have a ticket on this?

Reply:
@ilyachur I can't find it, there is probably no such ticket., could you please create it?

Reply:
Created a ticket -36145